### PR TITLE
[docs][virtualization] dvp admin install sync with GS

### PR DESCRIPTION
**Pull request by diafour**
## Description

Sync "Installation" pages with Getting Started.

## Changelog entries
```changes
section: docs
type: chore
summary: Sync DVP Installation pages with the Getting Started.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: docs/site/pages/virtualization-platform/documentation/admin/update/UPDATE_RU.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
 
 Платформа поддерживает три режима обновления:
 
-- **Автоматический + окна обновлений не заданы.** Кластер обновится сразу после появления новой версии на соответствующем [канале обновлений](../update-channels.html).
+- **Автоматический + окна обновлений не заданы.** Кластер обновится сразу после появления новой версии на соответствующем [канале обновлений](../release-channels.html).
```
Comment (z9r5) [inline docs/site/pages/virtualization-platform/documentation/admin/update/UPDATE_RU.md:35]: Please check link in eng version

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE.md [hunk 2]
```diff
@@ -232,23 +232,23 @@ Next, you will need to create an Ingress controller, a user to access the web in
...
+sudo -i d8 k -n d8-ingress-nginx get po -l app=controller
 ```
 {% endsnippetcut %}
 
 Wait for the Ingress controller pods to switch to <code>Ready</code> state.
 
 {% offtopic title="Example of the output..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get po -l app=controller
+sudo -i d8 k -n d8-ingress-nginx get po -l app=controller
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE.md:251]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get po -l app=controller
```

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 3]
```diff
@@ -193,13 +193,13 @@ EOF
...
 ```shell
-sudo /opt/deckhouse/bin/kubectl get no
+sudo -i d8 k get no
 ```
 {% endsnippetcut %}
 
 {% offtopic title="Пример вывода..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl get no
+sudo -i d8 k get no
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:202]:
```suggestion
$ sudo -i d8 k get no
```

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 4]
```diff
@@ -217,13 +217,13 @@ d8cluster-worker   Ready    worker                 10m   v1.23.17
...
 ```shell
-sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get po -l app=kruise
+sudo -i d8 k -n d8-ingress-nginx get po -l app=kruise
 ```
 {% endsnippetcut %}
 
 {% offtopic title="Пример вывода..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get po -l app=kruise
+sudo -i d8 k -n d8-ingress-nginx get po -l app=kruise
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:226]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get po -l app=kruise
```

File: docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md [hunk 5]
```diff
@@ -240,23 +240,23 @@ kruise-controller-manager-7dfcbdc549-b4wk7   3/3     Running   0           15m
...
+sudo -i d8 k -n d8-ingress-nginx get po -l app=controller
 ```
 {% endsnippetcut %}
 
 Дождитесь перехода подов Ingress-контроллера в статус <code>Ready</code>.
 
 {% offtopic title="Пример вывода..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get po -l app=controller
+sudo -i d8 k -n d8-ingress-nginx get po -l app=controller
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/bm/STEP_FINALIZE_RU.md:259]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get po -l app=controller
```

File: docs/site/_includes/getting_started/dvp/global/STEP_NODES_RU.md [hunk 6]
```diff
@@ -89,7 +89,7 @@ pdpl-user -i 63 caps
   {% snippetcut %}
   ```shell
 export NODE=<NODE-IP-ADDRESS> # Укажите IP-адрес узла, который необходимо подключить к кластеру.
-  kubectl create -f - <<EOF
+sudo -i d8 k create -f - <<EOF
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/dvp/global/STEP_NODES_RU.md:92]:
```suggestion
  sudo -i d8 k create -f - <<EOF
```

File: docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS.md [hunk 7]
```diff
@@ -16,13 +16,13 @@ ssh {% if page.platform_code == "azure" %}azureuser{% elsif page.platform_code =
...
 ```shell
-sudo /opt/deckhouse/bin/kubectl get nodes
+sudo -i d8 k get nodes
 ```
 {% endsnippetcut %}
 
 {% offtopic title="Example of the output..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl get nodes
+sudo -i d8 k get nodes
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS.md:25]:
```suggestion
$ sudo -i d8 k get nodes
```

File: docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS.md [hunk 8]
```diff
@@ -33,15 +33,15 @@ It may take some time to start the Ingress controller after installing Deckhouse
...
+sudo -i d8 k-n d8-ingress-nginx get po
 ```
 {% endsnippetcut %}
 
 Wait for the Pods to switch to `Ready` state.
 
 {% offtopic title="Example of the output..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get po
+sudo -i d8 k -n d8-ingress-nginx get po
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS.md:44]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get po
```

File: docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS.md [hunk 9]
```diff
@@ -52,15 +52,15 @@ kruise-controller-manager-78786f57-82wph   3/3     Running   0          16h
...
+sudo -i d8 k -n d8-ingress-nginx get svc nginx-load-balancer
 ```
 {% endsnippetcut %}
 
 The `EXTERNAL-IP` value must be filled with a public IP address or DNS name.
 
 {% offtopic title="Example of the output..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get svc nginx-load-balancer
+sudo -i d8 k -n d8-ingress-nginx get svc nginx-load-balancer
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS.md:63]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get svc nginx-load-balancer
```

File: docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS_RU.md [hunk 10]
```diff
@@ -16,13 +16,13 @@ ssh {% if page.platform_code == "azure" %}azureuser{% elsif page.platform_code =
...
 ```shell
-sudo /opt/deckhouse/bin/kubectl get nodes
+sudo -i d8 k get nodes
 ```
 {% endsnippetcut %}
 
 {% offtopic title="Пример вывода..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl get nodes
+sudo -i d8 k get nodes
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS_RU.md:25]:
```suggestion
$ sudo -i d8 k get nodes
```

File: docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS_RU.md [hunk 11]
```diff
@@ -33,15 +33,15 @@ cloud-demo-worker-01a5df48-84549-jwxwm   Ready    worker                 12h   v
...
+sudo -i d8 k -n d8-ingress-nginx get po
 ```
 {% endsnippetcut %}
 
 Дождитесь перехода Pod'ов в статус `Ready`.
 
 {% offtopic title="Пример вывода..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get po
+sudo -i d8 k -n d8-ingress-nginx get po
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS_RU.md:44]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get po
```

File: docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS_RU.md [hunk 12]
```diff
@@ -52,15 +52,15 @@ kruise-controller-manager-78786f57-82wph   3/3     Running   0          16h
...
+sudo -i d8 k -n d8-ingress-nginx get svc nginx-load-balancer
 ```
 {% endsnippetcut %}
 
 Значение `EXTERNAL-IP` должно быть заполнено публичным IP-адресом или DNS-именем.
 
 {% offtopic title="Пример вывода..." %}
 ```
-$ sudo /opt/deckhouse/bin/kubectl -n d8-ingress-nginx get svc nginx-load-balancer
+sudo -i d8 k -n d8-ingress-nginx get svc nginx-load-balancer
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/STEP_CLUSTER_ACCESS_RU.md:63]:
```suggestion
$ sudo -i d8 k -n d8-ingress-nginx get svc nginx-load-balancer
```

**Review by mortelumina98 — APPROVED** [review]

**Pull request merged**